src/abstractdataframe/abstractdataframe.jl

```diff
@@ -811,10 +815,14 @@ julia> dropmissing(df, [:x, :y])
 """
 function dropmissing(df::AbstractDataFrame,
                      cols::Union{ColumnIndex, MultiColumnIndex}=:;
-                     disallowmissing::Bool=true)
-    newdf = df[completecases(df, cols), :]
+                     view::Bool=false, disallowmissing::Bool=!view)
+    view && disallowmissing && throw(ArgumentError("it is not possible to disallow " *
+                                                   "missing when view=true"))
+    rowidxs = completecases(df, cols)
+    view && return Base.view(df, completecases(df, cols), :)
```

Comment:
```suggestion
    view && return Base.view(df, rowidxs, :)
```

Also, maybe it would be clearer with `if view... else`?